bug-bounty-hof/client.yml

```diff
@@ -1,4 +1,55 @@
 names:
+- name: Muhammad R. Maulana
+  date: 2020-12-21
+  twitter: "@agamimaulana"
+  url: https://twitter.com/@agamimaulana
+- name: Irvan Kurniawan (sourc7)
+  date: 2020-12-15
+  url: https://www.linkedin.com/in/sourc7/
+- name: Nimrod Aviram
+  date: 2020-12-08
+- name: Pete Freitag
+  date: 2020-12-08
+- name: Michał Bentkowski
+  date: 2020-12-08
+- name: Gary Kwong
+  date: 2020-12-08
+- name: Pham Bao of VinCSS (Member of Vingroup)
+  date: 2020-12-08
+- name: Omair and Abraruddin Khan
```

Comment:
Since these are not brothers either reverse the order or separate with a comma rather than "and"

Reply:
Tom says that's how they specified the credit line.